src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -127,12 +107,7 @@ const SetAclPermsDocContainerForm = () => {
           </FormControl>
           <br />
           <FormControl fullWidth>
-            <Button
-              variant="contained"
-              disabled={permissionState.podUrlToSetPermissionsTo === podUrl || state.processing}
```

Comment:
The previous disabling had the button also disabled it when the user put in their own podUrl - so they couldn't try to change permissions of their own pod.   Think we need to add this back in, unless you found another way to address that possibility.

Reply:
Added this back in.